selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -655,6 +656,22 @@ public void goToPosition(int positionLine, int positionChar) {
     waitCursorPosition(positionLine, positionChar);
   }
 
+  /**
+   * Sets cursor to specified {@code positionLine} and {@code positionChar} and checks result.
+   *
+   * @param positionLine line's number where cursor should be placed
+   * @param positionChar char's number where cursor should be placed
+   */
+  public void selectLines(int fromLine, int numberOfLine) {
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
+    setCursorToLine(fromLine);
+    action.keyDown(SHIFT).perform();
+    for (int i = 0; i < numberOfLine; i++) {
+      typeTextIntoEditor(Keys.ARROW_DOWN.toString());
+    }
+    action.keyUp(SHIFT).perform();
```

Comment:
I think you should add sending "END" button:
```
    action.sendKeys(Keys.END.toString()).keyUp(SHIFT).perform();
```

Reply:
Done